### DISCUSSION TITLE
Support BSC and HECO

### DISCUSSION THREAD

**Discussion by dawafisher** [DimensionDev/Maskbook#2564] [Ideas]
ETH is too expensive. Supporting BSC and HECO will be welcome.

**Comment by Tedko** [comment]
> ETH is too expensive. Supporting BSC and HECO will be welcome.

Hi there.

We're supporting multi chain. If you're a developer welcome to join our dev group!

**Reply by dawafisher** [reply]
Great！I hope I can join this exciting project but I don't have the professional skills. Mask Network will be a great product in Web 3.0

**Reply by bluabaleno** [reply]
What you mean by multi-chain support? Is that for storing content on multiple blockchains or supporting their tokens?

**Reply by PAKISTANECN1** [reply]
Есть отличные представления в развитии Mask она может быть как и хранилищем так и связью для твоих платформ , сайтов , социальных сетей - нужно только дорабатывать . Возможна вариации своего токена Маск ? нужно запросить спонсорство для разработки с этого и начнётся развитие и токена и полноценной платформы .
